e2e_tests/tests/api_utils.py

```diff
@@ -0,0 +1,10 @@
+from determined.common import api
+from determined.common.api import authentication, certs
+from tests import config as conf
+
+
+def determined_test_session() -> api.Session:
```

Comment:
heads up for a rebase conflict: I updated this method in https://github.com/determined-ai/determined/pull/5055/files#diff-66c234e5c973d0546d1f6713c5b496deca22124eea269b1f8c5a3fb0c1193c81R275